I'm getting a failed test when building locally and can't figure out why. Maybe you know what this is about and how to fix it?

```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.opencastproject.speechtotext.impl.SpeechToTextServiceImplTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.364 s <<< FAILURE! - in org.opencastproject.speechtotext.impl.SpeechToTextServiceImplTest
[ERROR] testVoskSpeechToText(org.opencastproject.speechtotext.impl.SpeechToTextServiceImplTest)  Time elapsed: 2.227 s  <<< ERROR!
java.nio.file.NoSuchFileException: /home/arnewilken/IdeaProjects/opencast13-2/modules/speech-to-text-impl/target/junit10331445475579306707/junit11583205952770915644/speech_to_text_test.vtt,eng
	at org.opencastproject.speechtotext.impl.SpeechToTextServiceImplTest.testVoskSpeechToText(SpeechToTextServiceImplTest.java:154)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   SpeechToTextServiceImplTest.testVoskSpeechToText:154 » NoSuchFile /home/arnewi...
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

--- a/docs/guides/admin/docs/workflowoperationhandlers/speechtotext-woh.md
+++ b/docs/guides/admin/docs/workflowoperationhandlers/speechtotext-woh.md
@@ -14,15 +14,15 @@ engines available, [Whisper](../modules/transcription.modules/whisper.md) and
 Parameter Table
 ---------------
 
-| configuration keys | required | description                                                                                                                                             |
-|--------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| source-flavor      | yes      | The source media package to use                                                                                                                         |
-| target-flavor      | yes      | Flavor of the produced subtitle file. The subflavor supports the language-code placeholder `#{lang}`                                                    |
-| target-element     | no       | Define where to append the subtitles file. Possibilities are: as a 'track' or as an 'attachment'. The default is "attachment".                          |
-| language-code      | no       | The language of the video or audio source (default is "eng"). Vosk only: It has to match the name of the language model directory. See 'vosk-cli'.      |
-| language-fallback  | yes*     | The fallback value if the dublin core/media package language field is not present.                                                                      |
-| target-tags        | no       | Tags for the subtitle file (Whisper only: Will add always the tag `lang:{code}` Where `code` can be by autodetection or from `language-code` parameter) |
- | translate          | no       | Transcription is translated into English (Whisper Only)                                                                                                 |
+| configuration keys | required | Example           | description                                                                                                                                        |
+|--------------------|----------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| source-flavor      | yes      | source/presenter  | The source media package to use                                                                                                                    |
+| target-flavor      | yes      | archive           | Flavor of the produced subtitle file. The subflavor supports the language-code placeholder `#{lang}`                                               |
+| target-element     | no       | attachment        | Define where to append the subtitles file. Possibilities are: as a 'track' or as an 'attachment'. The default is "attachment".                     |
+| language-code      | no       | de                | The language of the video or audio source (default is "eng"). Vosk only: It has to match the name of the language model directory. See 'vosk-cli'. |
+| language-fallback  | yes*     | en                | The fallback value if the dublin core/media package language field is not present.                                                                 |
+| target-tags        | no       | delivery/captions | Tags for the subtitle file (Whisper only: If no `language-code`,the tag `lang:{code}` will be auto generated)                                      |
+ | translate          | no       | true              | Transcription is translated into English, valid values `true` or `false` (Whisper Only)                                                            |
 
  *Vosk Only, default value can be modified on Vosk config file.
 

--- a/modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/SpeechToTextServiceImpl.java
+++ b/modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/SpeechToTextServiceImpl.java
@@ -36,6 +36,7 @@ import org.opencastproject.workspace.api.Workspace;
 
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang3.BooleanUtils;
 import org.osgi.service.component.ComponentContext;
 import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
@@ -140,14 +141,10 @@ public class SpeechToTextServiceImpl extends AbstractJobProducer implements Spee
     List<String> arguments = job.getArguments();
     String language = arguments.get(1);
     URI mediaFile = new URI(arguments.get(0));
-    Boolean translate = false;
-    try {
-      translate = Boolean.parseBoolean(arguments.get(2));
-    } catch (NumberFormatException e) {
-      throw new SpeechToTextServiceException("The translate parameter must be a boolean. Declared value: \""
-              + arguments.get(2) + "\"");
+    Boolean translate = BooleanUtils.toBooleanObject(arguments.get(2));
+    if (translate == null) {
+      translate = false;
     }
-
     URI subtitleFilesURI;
     File subtitlesFile = null;
     String vttFileName = String.format("%s%d_%s.%s", TMP_PREFIX,

--- a/modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java
+++ b/modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java
@@ -293,13 +293,17 @@ public class
     if (stringTranslateMode.isEmpty()) {
       translateMode = false;
     } else {
-      try {
-        translateMode = Boolean.parseBoolean(stringTranslateMode);
-      } catch (NumberFormatException e) {
-        throw new WorkflowOperationException(String.format(
+      if ("true".equals(stringTranslateMode) || "1".equals(stringTranslateMode)
+          || "yes".equals(stringTranslateMode)) {
+        translateMode = true;
+      } else if ("false".equals(stringTranslateMode) || "0".equals(stringTranslateMode)
+              || "no".equals(stringTranslateMode)) {
+        translateMode = false;
+      } else {
+        throw new IllegalArgumentException(String.format(
             "Speech-to-Text job for media package '%s' failed, Because invalid \"translate\" value ('%s')."
                 + "Valid values types: \"true\", \"1\", \"yes\", \"false\", \"0\", or \"no\".", mediaPackage,
-                stringTranslateMode));
+            stringTranslateMode));
       }
     }
     return translateMode;

